content/docs/start/data-and-model-versioning.md

```diff
@@ -140,12 +139,10 @@ set up earlier. You can check that the data has been stored in the DVC remote
 with:
 
 ```dvc
-$ ls -R /tmp/dvcstore
-/tmp/dvcstore/:
-a3
-
-/tmp/dvcstore/a3:
-04afb96060aad90176268345e10355
+$ tree /tmp/dvcstore
```

Comment:
I changed this to `tree` because that also exists in Windows `cmd` even when the output will be different -- should be close enough (haven't checked bc/ I'm away from my Windows machine for a few days).

But the issue with the non-Win `/` root folder persists thought out this doc.

Reply:
yeah could use something like %TEMP%\store or %USER%\dvc\store ?

Reply:
See the previous https://github.com/iterative/dvc.org/pull/3273#pullrequestreview-880425596